chromium_src/chrome/elevation_service/elevator.cc

```diff
@@ -0,0 +1,38 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_vpn/common/buildflags/buildflags.h"
+
+#if BUILDFLAG(ENABLE_BRAVE_VPN)
+#include "brave/components/brave_vpn/browser/connection/ikev2/win/brave_vpn_helper/brave_vpn_helper_utils.h"
+#include "brave/components/brave_vpn/common/wireguard/win/wireguard_utils_win.h"
+#include "brave/elevation_service/install_utils.h"
+#include "third_party/abseil-cpp/absl/cleanup/cleanup.h"
```

Comment:
headers should only go inside the buildflag if there is a matching guard in gn. This definitely applies to cleanup and probably also to elevation_service/install_utils.h